examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -57,6 +58,48 @@
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CHIPCommandBridge::GetPAACertsFromFolder(NSArray<NSData *> ** paaCertsResult)
```

Comment:
Should this have the `__autoreleasing` annotation on it as needed?

Reply:
Yes, good catch. Thank you!